### DISCUSSION TITLE
Shared folders hosted on another machine

### DISCUSSION THREAD

**Discussion by shanghaiultra** [slskd/slskd#861] [Q&A] [answered]
My docker host is on a VM hosted by Truenas, and the folders I want to share in slskd are located in my Truenas main system. So I am trying to get around this by sharing them via NFS network share and setting up a mount in /etc/ fstab of the docker host VM. 

So far the logs say my directory is invalid in the config, even although I can access this share from the docker host using the same path I have entered in the config.

From the configuration notes: "Sharing a root mount on a unix-like OS (/) is also not supported." - does this mean shared folders must be on the same machine as the docker installation? Are network shares supported?

**Comment by jpdillingham** [comment, marked as answer]
Network shares are supported (e.g. UNC paths), and mounts aren't treated any differently within the application; it's transparent.  Some of the earliest versions were developed using NFS mounts, so it's completely possible.  I did find them to be slow and unreliable, but that's not an application issue.

You might be missing a volume; I would suggest running the container interactively to see what mounts it knows about:

```bash
docker run -it --entrypoint /bin/sh slskd/slskd:latest
```

**Reply by shanghaiultra** [reply]
Seems that is the case, after entering the above, nothing happens and now the docker command itself results in "/bin/sh: 8: docker: not found" 

Somewhat out of my depth here - guessing that just because the docker host system can see a mount doesn't mean docker itself can? Any pointers?

**Reply by jpdillingham** [reply]
Sorry I neglected to follow up back in January.  Is this still an issue?

**Reply by shanghaiultra** [reply]
My turn to apologise for coming back late... the issue was down to my ignorance of docker. It was indeed a case of shares not being mounted properly.
